Functions/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v2.3.0
+- Change the default timeout for callable functions to 70s.
```

Comment:
Please add a PR number beside each (even if they are the same).

Reply:
Done.